clear startport autogen

dzmitryhil opened the pull request:
- remove legacy unused files
- add PHONY to makefile
- remove auto-generated release.yml
- rename gaia -> onomy

sunnyk56 commented:
All work of this PR has been updated in the single PR https://github.com/onomyprotocol/onomy/pull/16 , so we are close to this PR.

dzmitryhil committed:
- remove legacy unused files
- add PHONY to makefile
- remove auto-generated release.yml
- rename gaia -> onomy
